index.html

```diff
@@ -8,47 +8,355 @@
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="technology">
+                <ul>
+                  <li><a href="#"></a>html</li>
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+          <!-- ----------card 4---------- -->
+          <div class="content_2" id="card_4">
+            <img src="/images/Img1.png" alt="Mucyo" class="image" />
+            <div class="card">
+              <h1>Profesonal Art <br />Printing Data</h1>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="technology">
+                <ul>
+                  <li><a href="#"></a>html</li>
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+          <div class="content_2" id="card_5">
+            <img src="/images/Img3.png" alt="Mucyo" class="image" />
+            <div class="card">
+              <h1>Profesonal Art <br />Printing Data</h1>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="technology">
+                <ul>
+                  <li><a href="#"></a>html</li>
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+
+          <div class="content_2" id="card_6">
+            <img src="/images/Img4.png" alt="Mucyo" class="image" />
+            <div class="card">
+              <h1>Profesonal Art <br />Printing Data</h1>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="technology">
+                <ul>
+                  <li><a href="#"></a>html</li>
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+        </div>
+        <div id="content_one">
+          <div class="content_3">
```

Comment:
> The about section is being overflowed by the projects section. Give it a fix.
![Screenshot from 2021-08-20 12-37-06](https://user-images.githubusercontent.com/65501535/130213806-1c1fc97e-b265-4916-a620-90043f50d3c6.png)

Please consider fixing the overflow issue